pages/common/qmmp.md

```diff
@@ -0,0 +1,32 @@
+# qmmp
+
+> A small and light audio player.
+> More information: <http://qmmp.ylsoftware.com/>.
```

Comment:
```suggestion
> More information: <https://manned.org/qmmp>.
```
Can we use manned or any official documentation link here instead?

Reply:
Is it not the official page? From what i can see it is.

Reply:
> Is it not the official page? From what I can see it is.

Yes, It is an official page but I couldn't find the documentation link in it. If it is present can you link it to it here?

Reply:
We should consider that the manned is last resort if there is no homepage for the command, I think it is fine, but let's see what others think

Reply:
```suggestion
> More information: <http://qmmp.ylsoftware.com>.
```